cmd/cycloid/kpis/list-available.go

```diff
@@ -42,17 +42,21 @@ func listAvailable(cmd *cobra.Command, args []string) error {
 		return errors.Wrap(err, "unable to get output flag")
 	}
 
-	kpis, err := m.ListAvailableKpi(org)
-	if err != nil {
-		return err
-	}
-
 	// fetch the printer from the factory
 	p, err := factory.GetPrinter(output)
 	if err != nil {
 		return errors.Wrap(err, "unable to get printer")
 	}
 
+	kpis, err := m.ListAvailableKpi(org)
```

Comment:
[getAvailableKPIs](https://github.com/cycloidio/youdeploy-http-api/pull/2603) is going to be removed, maybe we can remove it right away.